tests/unit/test_parser.py

```diff
@@ -85,30 +86,30 @@ def test_parser(self):
     def test_triples(self):
         speed = calculate_speed()
         data = get_inputs_and_outputs(speed)
-        graph = get_triples(data=data, NS=NS)
+        graph = get_triples(data=data)
         self.assertGreater(
-            len(list(graph.triples((None, NS.hasUnits, NS["meter/second"])))), 0
+            len(list(graph.triples((None, PNS.hasUnits, URIRef("meter/second"))))), 0
         )
-        ex_triple = (None, NS.isOutputOf, NS["calculate_speed.inputs.time"])
+        ex_triple = (None, EX.isOutputOf, URIRef("calculate_speed.inputs.time"))
```

Comment:
Great idea. I was wondering to what degree this shoehorns other users/developers, so these examples with the `EX` namespace are super helpful for my understanding.

In this one place, shouldn't `isOutputOf` be part of the fixed pyiron ontology since it's needed for interpreting all ontologies in the framework of a workflow graph?

Reply:
Here it was only testing whether a triple can be correctly parsed and anything would have been fine. But it's true that for maintenance reasons it should be clearer